synapse/storage/databases/main/user_erasure_store.py

```diff
@@ -40,7 +40,7 @@ async def is_user_erased(self, user_id: str) -> bool:
         return bool(result)
 
     @cachedList(cached_method_name="is_user_erased", list_name="user_ids")
-    async def are_users_erased(self, user_ids: Iterable[str]) -> Dict[str, bool]:
+    async def are_users_erased(self, user_ids: Collection[str]) -> Dict[str, bool]:
```

Comment:
this can be an Iterable now

Reply:
Arg... sorry I should have reverted that commit. 😢